draad-kaarten.php

```diff
@@ -3,6 +3,7 @@
 /**
  * Plugin Name: Draad Kaarten
  * text-domain: draad
```

Comment:
No description will show empty cell in the plugins table, probably best to add a short summary of the plugin functionality